### DISCUSSION TITLE
Small contribution

### DISCUSSION THREAD

**Discussion by AgroMeester** [Mogli12/KeyboardSteer#734] [General]
Hi Stefan,

Just wanted to make a small contribution to your outstanding work. It's a new store icon for KeyboardSteer. Use if you like!

Greetings,

AgroMeester

![keyboardsteer](https://user-images.githubusercontent.com/25507803/50588983-e4496d00-0e84-11e9-88e1-62df4e089324.png)

**Comment by Mogli12** [comment]
Hi AgroMesster,

This is really nice. Thanks a lot.

Best regards,
Stefan

**Comment by AgroMeester** [comment]
Glad to hear/see you like it Stefan!

I've added a transparant version of the image. Maybe this one blends nicer with the standard background of the store?

Greetings,

AgroMeester

![keyboardsteer-v2](https://user-images.githubusercontent.com/25507803/50602333-6488c600-0eb7-11e9-9b44-6580f0b7ba3c.png)

**Comment by Mogli12** [comment]
Hi AgroMeester,
Can you directly convert it into 256x256? If I do it it does not look so nice anymore.
Thanks, Stefan

**Comment by AgroMeester** [comment]
Hi Stefan,

Sure, no problem.

Here you go!

![keyboardsteer-v3](https://user-images.githubusercontent.com/25507803/51186385-cd216b00-18d9-11e9-9486-45dfdcd9ce55.png)

**Comment by Mogli12** [comment]
Hi,

Thank you. That looks great.

Regards,
Stefan

**Comment by AgroMeester** [comment]
Hi Stefan,

I like the HUD change you made from 'F' / 'R' to corresponding arrows which made me think up some HUD options myself. If you're interested, I can help you with the graphics.

Greetings,

AgroMeester

 
![gui-v1](https://user-images.githubusercontent.com/25507803/51209089-b39b1600-190f-11e9-9020-1d3a29dbe8e5.png)

**Comment by Mogli12** [comment]
Project is stopped as it does not work together with sleep on Dedi server.

**Comment by AgroMeester** [comment]
![vehiclecontroladdon-v1](https://user-images.githubusercontent.com/25507803/52337878-2710de80-2a09-11e9-82ed-a85725875064.png)

Hi Stefan,

I saw you were working on this mod again so here's a little treat.

**Comment by Mogli12** [comment]
Really nice.
